wiki/en/Privacy-Statement.md

```diff
@@ -12,8 +12,8 @@ permalink: "/wiki/Privacy-Statement"
 
 ## Definition of Terms
 
-- "**Server**" The Jamulus Server software running on a remote computer to which you will connect.
-- "**Client**" The Jamulus software running on your local computer used to connect to a Server
+- "**Server**" The Jamulus Server software running on a remote computer to which Clients connect
+- "**Client**" The Jamulus software used to connect to a Server
```

Comment:
The above two lines make it sound like there are two different pieces of software.  That's not _necessarily_ the case.

Reply:
Given that this is just a definition for the purposes of the privacy statement, could we say:

"Server" An instance of Jamulus configured as a Server for Clients
"Client"  An instance of Jamulus configured to connect to Jamulus configured as a Server. 

I note that the man page refers to "server mode" and "client mode" - maybe that's easier to understand in this context?

"Server" An instance of Jamulus configured in server mode.
"Client"  An instance of Jamulus configured client mode.

Reply:
See [@mcfnord's suggestion](https://github.com/jamulussoftware/jamuluswebsite/pull/872#pullrequestreview-1178170645).